ui/src/components/Configuration/RequisitionDefinitionsLayout.vue

```diff
@@ -60,24 +62,26 @@ import { useStore } from 'vuex'
 import router from '@/router';
 import { notify } from "@kyvg/vue3-notification"
 import { putProvisionDService } from "./../../services/configurationService"
+import { FeatherSortObject } from '@/types'
 
 const store = useStore()
 const buttonAction = ref(['ADD NEW', 'BACK'])
 const index = ref(0)
 const icon = ref(markRaw(actionsAdd));
-const isData = ref(false)
+const isData = ref(false);
+const sortObj = ref({'value':'asc','property':'import-name'});
+const sortableColms = ref([{'value':'asc','property':'import-name'}, {'value':'none','property':'cron-schedule'}, {'value':'none','property':'import-url-resource'}, {'value':'none','property':'rescan-existing'}]);
```

Comment:
Can we break this line so it's not so long?
ref([
{value:'asc',property:'import-name'},
...
])

Reply:
done